cmd/chainbridge/account.go

```diff
@@ -97,13 +75,40 @@ func handleGenerateCmd(ctx *cli.Context) error {
 		}
 	}
 
-	_, err = generateKeypair(keytype, datadir, password, privKey)
+	_, err := generateKeypair(keytype, dHandler.datadir, password, privKey)
 	if err != nil {
 		return fmt.Errorf("failed to generate key: %s", err)
 	}
 	return nil
 }
 
+// handleImportCmd imports external keystores into the bridge
+func handleImportCmd(ctx *cli.Context, dHandler *dataHandler) error {
+
+	// import key
+	if keyimport := ctx.Args().First(); keyimport != "" {
+		log.Info("Importing key...")
+		_, err := importKey(keyimport, dHandler.datadir)
+		if err != nil {
+			return fmt.Errorf("failed to import key: %s", err)
+		}
+	}
```

Comment:
Please add an else that returns an error along the lines of "must provide a key"